File: Python/chandu/Kth_Smallest_Element_In_BST.py
def solve(root, k):
    # CODE HERE
    # using inorder transversal
    kat = []
    def inorder(root):
        if not root:
            return []
        inorder(root.left)
        kat.append(root.val)
        inorder(root.right)

        return kat
    
    return inorder(root)[k-1]

File: java/mohit/Trees-BST/Kth Smallest Element In BST.java
static int counter = 0;
static int ans = 0;
static int solve(TreeNode root, int k)
{
    if (root==null) return 0;
    
    solve(root.left, k);

    counter ++;
    if(counter == k)    ans = root.val;

    if(counter < k) solve(root.right, k);

    return ans;
}